packages/devtools_server/docs/daemon.md

```diff
@@ -0,0 +1,106 @@
+        'host': 'localhost',
+        'port': 9100,
+        'pid': 9876,
+    }
+}
+```
+
+#### server.started Event
+
+This event is sent when the server starts up and has bound to a port. Each event contains the following params:
+
+- `host` - the hostname/IP the server is listening on
+- `port` - the port the server bound to (this may differ from the supplied `--port` if `--try-ports` was used)
+- `pid` - the PID of the current process (supplied to improve reliability of sending kill signals since the process handle the IDE has may be a shell)
+- `protocolVersion` - the version number for this protocol (if this field is missing it is protocol version `1.0.0`)
+
+<!--
+This request is only used for testing purposes so is currently "undocumented"
+
+### client.launch Event
+
+This event is sent when the server launches a new client in response to a call to `launchDevTools`. `params` contains the following fields:
+
+- `reused` - whether an existing DevTools instance was reused (otherwise a new browser was launched)
+- `notified` - whether or not a notification was shown
+-->
+
+### Requests
+
+### `vm.register` Request
+
+This request is used to tell the server about a new VM service it should connect to and register the `launchDevTools` service into. The request takes the following params:
+
+- `uri` - the URI of the VM service to register with
+
+<!--
+This request is only used for testing purposes so is currently "undocumented"
+
+### `client.list` Request
+
+This request lists all DevTools instances that are currently connected back to the server along with which VM services they're connected to and the pages they are showing. The request requires no `params`.
+-->
+
+### launchDevTools Request (VM Service)
+
+`launchDevTools` is registered as a VM service (it is *not* sent over stdin to the server) and takes the following params:
+
+- `reuseWindows` - whether an existing DevTools instance that is not connected to a VM (or is connected to the same one) should be reused
+- `notify` - whether to send a browser notification to the user in the case where a DevTools instance is reused, to help them find the window
+- `page` - the page to launch DevTools on (matches the IDs used in DevTools that show in the URL fragments) or - if reusing a window - to switch to
+- `queryParams` - an object of additional query parameters that may be added to the query string that may influence DevTools behaviour, such as:
+  - `theme` - allows using the `dark` theme
+  - `ide` - the client (eg. `VSCode`) to be logged in analytics
+  - `hide` - IDs of pages to hide (eg. `debugger` when launching from an IDE with its own debugger)
+
+#### Example
+
+```js
+{
+    "id": "123",
+    // The `method` field is populated based on the ServiceRegistered VM event
+	"method": "s2.launchDevTools",
+	"params": {
+		"notify": true,
+		"page": "inspector",
+		"queryParams": {
+			"hide": "debugger",
+			"ide": "VSCode",
+			"theme": "dark"
+		},
+		"reuseWindows": true
+	}
```

Comment:
nit: use single quotes and make sure indent is consistent in json examples

Reply:
Oops, both done. I had a mix of tabs and spaces that happened to line up in my editor 😔